src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPanelAdapter.java

```diff
@@ -4,21 +4,25 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+import edu.kis.powp.jobs2d.features.DriverFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawPanelAdapter implements Job2dDriver {
```

Comment:
The name lacks 'driver' aspect. (Adapter is only an implementation detail)

Reply:
Fixed